ChatSecure/Classes/View Controllers/OTRMessagesGroupViewController.m

```diff
@@ -44,6 +44,11 @@ - (void)setupGroupChat:(NSArray <NSString *>*)buddies account:(OTRAccount *)acco
     [self setThreadKey:self.threadKey collection:[OTRXMPPRoom collection]];
 }
 
+- (nullable OTRAccount *)accountWithTransaction:(nonnull YapDatabaseReadTransaction *)transaction {
```

Comment:
Ahhhh ok, I missed that because GitHub does a good job of hiding the extended commit messages. Perhaps add an inline comment that this is overriding super's behavior bc of that?

Btw it looks like your computer's date is wrong, says these commits are from April 12?